labellab_mobile/lib/screen/history/history_search_screen.dart

```diff
@@ -0,0 +1,164 @@
+                    )
+                  : Container(),
+              state.classifications != null
+                  ? Column(
+                      children: state.classifications
+                          .where((classification) {
+                            bool found = false;
+                            classification.label.forEach((label) {
+                              if (label.name
+                                  .toLowerCase()
+                                  .contains(query.toLowerCase())) {
+                                found = true;
+                              }
+                            });
+                            return found;
+                          })
+                          .map((classification) => HistoryItem(
+                                classification,
+                                onDeleteSelected: () {
+                                  _showOnDeleteAlert(context, classification);
+                                },
+                                onSelected: () => _gotoClassification(
+                                    context, classification.id),
+                              ))
+                          .toList(),
+                    )
+                  : Container(),
+            ],
+          )
+        : EmptyPlaceholder(
+            description: "Your past classifications will appear here",
+          );
+  }
+
+  @override
+  Widget buildSuggestions(BuildContext context) {
+    return state.classifications != null && state.classifications.isNotEmpty
+        ? ListView(
+            children: <Widget>[
+              state.isLoading
+                  ? LinearProgressIndicator(
+                      backgroundColor: Theme.of(context).canvasColor,
+                    )
+                  : Container(
+                      height: 6,
+                    ),
+              state.error != null
+                  ? ListTile(
+                      title: Text(state.error),
+                    )
+                  : Container(),
+              state.classifications != null
+                  ? Column(
+                      children: state.classifications
+                          .where((classification) {
+                            bool found = false;
+                            classification.label.forEach((label) {
+                              if (label.name
+                                  .toLowerCase()
+                                  .contains(query.toLowerCase())) {
+                                found = true;
+                              }
+                            });
+                            return found;
+                          })
+                          .map((classification) => HistoryItem(
+                                classification,
+                                onDeleteSelected: () {
+                                  _showOnDeleteAlert(context, classification);
+                                },
```

Comment:
Remove the delete option from suggestions

Reply:
Ok will do that